### PR TITLE
Balder/wakeuptrigger java

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR

### CODE REVIEW

File: container-messagebus/src/main/resources/configdefinitions/container.jdisc.container-mbus.def [hunk 1]
```diff
@@ -15,6 +15,9 @@ numthreads int default=2
 # Optimize for latency, or throughput.
 optimize_for enum {LATENCY, THROUGHPUT} default=LATENCY
 
+# Number of writeevents before triggering wakeup of network thread.
+wakeup_trigger_count int default=1
```
Comment (havardpe) [inline container-messagebus/src/main/resources/configdefinitions/container.jdisc.container-mbus.def:19]: transport_events_before_wakeup

**Review by havardpe — COMMENTED** [review]
we should probably also start using a Params object in both Java and C++, probably with delayed resolving of crypto engine (and resolver for C++)

**Comment by havardpe** [comment]
this way looks good, I think we should do it like this in C++ as well

File: container-messagebus/src/main/resources/configdefinitions/container.jdisc.container-mbus.def (hunk 1 above)
Comment (baldersheim) [reply container-messagebus/src/main/resources/configdefinitions/container.jdisc.container-mbus.def]: Fixed

**Comment by baldersheim** [comment]
I will followup and complete c++ in the same pattern.
Then I will do parameter cleanup in both.

**Pull request merged**